woocommerce-services.php

```diff
@@ -1205,10 +1205,43 @@ public function add_meta_boxes() {
 			if ( $this->shipping_label->should_show_meta_box() ) {
 				add_meta_box( 'woocommerce-order-label', __( 'Shipping Label', 'woocommerce-services' ), array( $this->shipping_label, 'meta_box' ), null, 'side', 'default' );
 			}
+
+			add_meta_box( 'woocommerce-services-shipping-debug', __( 'Shipping Debug', 'woocommerce-services' ), array( $this, 'shipping_rate_packaging_debug_log_meta_box' ), 'shop_order', 'normal', 'default' );
+		}
+
+		public function shipping_rate_packaging_debug_log_meta_box( $post ) {
+			$order            = wc_get_order( $post );
+			$shipping_methods = $order->get_shipping_methods();
+
+			foreach ( $shipping_methods as $method ) {
+				if ( empty( $method['wc_connect_packing_log'] ) ) {
+					continue;
+				}
+
+				$method_instance = new WC_Connect_Shipping_Method( $method->get_instance_id() );
+			?>
+				<p>
+					<strong>Shipping Method:</strong> <?php echo $method_instance->get_method_title(); ?>
+				</p>
+				<p>
+					<strong>Instance: </strong>
```

Comment:
Language nitpick here - "Instance" might not mean much to users. How about using "Shipping Method ID" here, and "Shipping Method Name:" in the line above instead of "Shipping Method"?